### DISCUSSION TITLE
How to Delete a list data in cases that we make

### DISCUSSION THREAD

**Discussion by stmarya** [Security-Onion-Solutions/securityonion#11452] [Unsupported Versions] [answered]
we have input some data and we need to remove it from the list
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/136021057/2a13d1e2-8e6d-40b2-a3da-511b622e6e0d)

**Comment by dougburks** [comment, marked as answer]
Answered here:
https://github.com/Security-Onion-Solutions/securityonion/discussions/11388#discussioncomment-7164268
